webUI/react/src/components/ReaderNavBar.tsx

```diff
@@ -182,6 +183,11 @@ export default function ReaderNavBar(props: IProps) {
             setHideOpenButton(currentScrollPos > prevScrollPos);
             setPrevScrollPos(currentScrollPos);
         }
+        if (settings.loadNextonEnding && (settings.readerType === 'ContinuesVertical' || settings.readerType === 'Webtoon')) {
+            if ((window.innerHeight + window.scrollY) >= document.body.offsetHeight) {
+                window.location.href = `/manga/${manga.id}/chapter/${chapter.index + 1}`;
+            }
+        }
```

Comment:
Each reader should provide some functionality based on the context, put this in `VerticalReader`

Reply:
I have moved this to the Vertical reader, but now the gradle check is not working even though I can compile everything locally.

Reply:
I made some changes, your code is not merge-able any more.

Reply:
> I made some changes, your code is not merge-able any more.

actually is, surprising!